### PR TITLE
Issue-#170 Add plugin for extracting CCSDS packets from AOS packets

### PR DESCRIPTION
**Pull request by Mejiro-McQueen**
+ Need plugin to extract CCSDS pakcets from AOS packets.

@kmarwah Still looks good to you?

### CODE REVIEW

File: ait/dsn/plugins/AOS_to_CCSDS.py [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+        super().__init__(inputs, outputs, zmq_args)
+        self.bytes_from_previous_frame = None
+
+    def process(self, data, topic=None):
+        AOS_frame_object = AOSTransFrame(data)
+        if AOS_frame_object.is_idle_frame:
+            log.debug(f"Dropping idle frame!")
+            return
+        else:
+            first_header_pointer = AOS_frame_object.get('mpdu_first_hdr_ptr')
```
Comment (nttoole) [inline ait/dsn/plugins/AOS_to_CCSDS.py:19]: Should there be a check to ensure the AOS frame is M_PDU type before processing ?

**Comment by nttoole** [comment]
@Mejiro-McQueen Could you also please include a quick doc page for this plugin?  Thanks

File: ait/dsn/plugins/AOS_to_CCSDS.py (hunk 1 above)
Comment (Mejiro-McQueen) [reply ait/dsn/plugins/AOS_to_CCSDS.py:21]: @kmarwah I added a check for M_PDU type into your commit.  Can you pull this branch and do a quick test with the fsws? It's down on my end.

**Comment by Mejiro-McQueen** [comment]
Added doc rst.

**Review by nttoole — CHANGES_REQUESTED** [review]
Last thing, could you please remove the commented out printf's?  (as a dev, I would prefer to keep them around at times - but it does add clutter).  Thanks

**Comment by Mejiro-McQueen** [comment]
@kmarwah  Let me know if you want me to do this for you.

**Review by nttoole — APPROVED** [review]

**Pull request merged**